src/test/resources/alma/HT016709661.json

```diff
@@ -129,6 +107,18 @@
     "id" : "http://lobid.org/resources/HT014176012#!",
     "type" : [ "Collection" ]
   } ],
+  "contribution" : [ {
+    "role" : {
+      "id" : "http://id.loc.gov/vocabulary/relators/ctb",
+      "label" : "Mitwirkende"
+    },
+    "type" : [ "Contribution" ],
+    "agent" : {
+      "label" : "Düsseldorf",
+      "type" : [ "CorporateBody" ],
+      "id" : "https://d-nb.info/gnd/$1"
+    }
```

Comment:
Assuming this is the issue mentioned in https://github.com/hbz/lobid-resources/issues/1155, right?

Reply:
yes